mobify/progressive-web-scaffold
# Cart Add to Wishlist functionality

This PR adds the ability to add to wishlist from the cart page. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1110
 **Linked PRs**: n/a

## Changes
- Added form data from PDP to the redux store
- Added modal asking guest users to sign in to add to wishlist
- Set up add to wishlist form submission
- Remove item from cart after successful add to wishlist submission
- Updated global fetch to correctly throw errors not related to fetching

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add one or more items to the cart
- Click "Save for later" for an item on the cart page
- If logged in: 
    - A modal will show saying the item was added
    - View the wishlist page to confirm the item was added
- If not logged in:
    - A modal will show prompting you to log in
